cypress/integration/App.spec.js

```diff
@@ -6,28 +6,30 @@ describe('App', () => {
     // Stub responses for getting and posting expense items
     cy.intercept('GET', '/items*', []);
     cy.intercept('POST', '/items*', {
-      id: 5,
-      title: 'Tasty test tacos',
-      date: '2022-02-01',
-      amount: 1499,
+      id: 6,
+      title: 'Test Expense - Cypress',
+      date: '2022-03-01T22:00:00.000Z',
+      amount: 499,
     });
 
     cy.visit('/');
-    cy.findByRole('textbox', { name: /expense name/i }).type('Test Tacos');
+    cy.findByRole('textbox', { name: /expense name/i }).type(
+      'Test Expense - Cypress'
+    );
```

Comment:
does this line wrap improve readability?

Reply:
😢 freaking prettier

Reply:
should we adapt the rules?

Reply:
I suggest we open a new issue and decide on a set of rules and the way to implement them with or without prettier @motine @MatthewBerglund

Reply:
we can do that. i would also be fine with adjusting the rules as we go